scripts/test_orchestration.sh

```
@@ -20,19 +20,19 @@ PASSED_TESTS=0
+    printf "%b%s\n" "${BLUE}[INFO]${NC} " "$1"
 }
 
 log_success() {
-    echo -e "${GREEN}[PASS]${NC} $1"
+    printf "%b%s\n" "${GREEN}[PASS]${NC} " "$1"
 }
 
 log_warning() {
-    echo -e "${YELLOW}[WARN]${NC} $1"
+    printf "%b%s\n" "${YELLOW}[WARN]${NC} " "$1"
 }
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)